security/experimental_authenticators.rst

```diff
@@ -502,3 +502,39 @@ authenticator, you would initialize the passport like this::
             ]);
         }
     }
+
+.. tip::
+
+    Besides badges, passports can also have generic attributes. This is
+    useful if you want to access information from ``authenticate()`` in
+    another authenticator method (e.g. ``createAuthenticatedToken()``)::
+
+        // ...
+        use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+
+        class LoginAuthenticator extends AbstractAuthenticator
+        {
+            // ...
+
+            public function authenticate(Request $request): PassportInterface
```

Comment:
Use statement for `PassportInterface` mising

Reply:
Yes, they are "collapsed" in the `// ...` comment. I wanted to make this code example as short as possible (because it's not a much needed feature I think)